### PR TITLE
[SYCL][ESIMD] Move ceil function into functional/common.hpp

### PR DESCRIPTION
**Pull request by vasilytric**
Because we are going to use it in different files

### CODE REVIEW

**Comment by vasilytric** [comment]
@sndmitriev Would you mind to review this PR?
Here just moving a function between two headers

**Review by sndmitriev — DISMISSED** [review]

**Comment by vasilytric** [comment]
@kbobrovs Would you mind to merge this PR?

File: SYCL/ESIMD/api/functional/common.hpp [hunk 1]
```diff
@@ -55,6 +55,10 @@ using shared_allocator = sycl::usm_allocator<DataT, sycl::usm::alloc::shared>;
 template <typename DataT>
 using shared_vector = std::vector<DataT, shared_allocator<DataT>>;
 
+constexpr int ceil(int a, int b) {
```
Comment (kbobrovs) [inline SYCL/ESIMD/api/functional/common.hpp]: please add description

**Review by kbobrovs — CHANGES_REQUESTED** [review]

File: SYCL/ESIMD/api/functional/common.hpp (hunk 1 above)
Comment (kbobrovs) [reply SYCL/ESIMD/api/functional/common.hpp]: BTW, here is the adopted definition of `ceil` function:
The ceil() function returns the smallest integer value that is greater than or equal to a number.

this `ceil` does something different, so please also change the name

Comment (vasilytric) [reply SYCL/ESIMD/api/functional/common.hpp]: The my `ceil` function has been renamed and the description has been added in this commit: https://github.com/intel/llvm-test-suite/pull/931/commits/f78dac9bb1254cf151c0714dad7700561ac03640

**Review by kbobrovs — APPROVED** [review]

**Comment by vasilytric** [comment]
@kbobrovs CI have passed, would you mind to merge this PR?

**Pull request merged**